src/simple-length.js

```diff
@@ -67,6 +67,25 @@
     return new SimpleLength((this.value - subtractedLength.value), this.type);
   };
 
+  SimpleLength.prototype._asCalcLength = function() {
+    var calcDictionary = {};
+
```

Comment:
These blank lines are unnecessary. I think it's good to use whitespace to break up blocks (exactly when is a matter of taste), but for very small blocks you should just bunch them up.